jwst/model_blender/blendmeta.py

```diff
@@ -233,6 +233,9 @@ def get_blended_metadata(input_models, verbose=False):
     else:
         new_table = None
 
+    for model in input_models:
+        model.close()
+
```

Comment:
Added this to prevent warnings about unclosed files at the end of the pipeline processing.